simple/lint.go

```diff
@@ -478,14 +478,18 @@ func negate(expr ast.Expr) ast.Expr {
 			Op: token.NOT,
 			X:  expr,
 		}
-	default:
-		return &ast.UnaryExpr{
-			Op: token.NOT,
-			X: &ast.ParenExpr{
-				X: expr,
-			},
+	case *ast.UnaryExpr:
+		if expr.Op == token.NOT {
+			return expr.X
 		}
 	}
+	return &ast.UnaryExpr{
```

Comment:
Please keep this as the `default` branch, and make the `case *ast.UnaryExpr` branch cover all of its cases.

Reply:
Done